packages/jupyterlab-manager/src/index.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+import '@fortawesome/fontawesome-free/css/all.css';
+import '@fortawesome/fontawesome-free/css/v4-shims.css';
+
```

Comment:
I wasn't sure where to put it, I'm of course open to better suggestions

Reply:
We are importing other css in plugin.ts. Can we move these imports to there?